examples/tutorial/tutorial_notebooks/introduction/omnisci_on_native/exercise_2.ipynb

```diff
@@ -0,0 +1,477 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Are they equals?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modin_df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for exercise: Reduces\n",
+    "\n",
+    "In pandas, a reduce would be something along the lines of a `sum` or `count`. It computes some summary statistics about the rows or columns. We will be using `sum`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "pandas_sum = (pandas_df[:4000000]).sum()\n",
+    "\n",
+    "end = time.time()\n",
+    "pandas_duration = end - start\n",
+    "\n",
+    "print(\"Time to count with pandas: {} seconds\".format(round(pandas_duration, 3)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "modin_sum = (modin_df[:4000000]).sum()\n",
+    "\n",
+    "end = time.time()\n",
+    "modin_duration = end - start\n",
+    "print(\"Time to count with Modin: {} seconds\".format(round(modin_duration, 3)))\n",
+    "\n",
+    "print(\"Modin is {}x faster than pandas at `sum`!\".format(round(pandas_duration / modin_duration, 2)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Are they equals?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas_sum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modin_sum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for exercise: Apply over a single column\n",
+    "\n",
+    "Sometimes we want to compute some summary statistics on a single column from our dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "rounded_trip_distance_pandas = pandas_df[\"trip_distance\"].apply(round)\n",
+    "\n",
+    "end = time.time()\n",
+    "pandas_duration = end - start\n",
+    "print(\"Time to groupby with pandas: {} seconds\".format(round(pandas_duration, 3)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "rounded_trip_distance_modin = modin_df[\"trip_distance\"].apply(round)\n",
```

Comment:
`Apply ` doesn't work on omnisci (defaulting to pandas) - do we really want it here?

Reply:
I deleted these lines.